docs/eta-pi-p/manual.ipynb

```diff
@@ -55,7 +52,34 @@
    "source": [
     "import matplotlib.pyplot as plt\n",
     "import numpy as np\n",
-    "import sympy as sp"
+    "import pandas as pd\n",
+    "import sympy as sp\n",
+    "from ampform.io import aslatex\n",
+    "from ampform.kinematics.angles import (\n",
+    "    Phi,\n",
+    "    Theta,\n",
+    ")\n",
```

Comment:
It seems to me that this would have fit on one line (within 88 characters). You can automate that with Ruff by setting [`split-on-trailing-comma`](https://docs.astral.sh/ruff/settings/#lint_isort_split-on-trailing-comma) in `pyproject.toml`:

```toml
[tool.ruff.lint.isort]
split-on-trailing-comma = false
```